requirements.txt

```diff
@@ -12,3 +12,9 @@ Flask-SQLAlchemy==2.3.2
 colorama==0.3.9
 colour==0.1.5
 psycopg2-binary==2.7.5
+webargs==4.0.0
+Werkzeug==0.14.1
+pytest==3.7.2
+pytest-runner==4.2
+alembic==1.4.2
```

Comment:
Alembic is used to manage migrations on devicehub-teal. Are you sure that is a requirement here?

Reply:
The service tag have models and the view save tags in the database.
The database is other of the database of dh. 
I suppose that in the future maybe we need to do a migration of this. And yes I think is a good idea have Alembic for this task

Reply:
Yes, I agree with you. But we don't have any related code to alembic yet, so it's not a current requirement.

Reply:
Remember, we want to keep our code and our project as clean and tiny as posible.